scripts/gen_FCHL19_rep.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python3
+import argparse
+import os
+import sys
+
+import numpy as np
+from ase.io import read, write
+from asaplib.io import str2bool
+from qml.representations import generate_fchl_acsf
+
+def repr_wrapper(frame, elements, is_periodic = False,
+                                        nRs2=24, nRs3=20,
+                                        nFourier=1, eta2=0.32, eta3=2.7,
+                                        zeta=np.pi, rcut=8.0, acut=8.0,
+                                        two_body_decay=1.8, three_body_decay=0.57,
+                                        three_body_weight=13.4):
+    '''
+   Periodic systems not implemented for FCHL19.
+   
```

Comment:
I believe Pylint should catch formatting-based things like the extra line here and the absence of a space between comma-separated function arguments. Would it be possible to include a docstring describing what the arguments are?

Reply:
Felix, we then need to add instructions on how to install/import qml.